src/Covid19Dashboard/index.jsx

```diff
@@ -159,8 +190,8 @@ class Covid19Dashboard extends React.Component {
               <div className='covid19-dashboard_visualizations'>
                 <WorldMapChart {...this.props} />
                 <div className='covid19-dashboard_charts'>
-                  {/* {seirChart} */}
-                  <div /> {/* TODO remove. just need to take some space */}
+                  {seirChart}
```

Comment:
can you please move the SEIR chart to the Illinois tab

Reply:
Moved. 👍